payment_paybox/__openerp__.py

```diff
@@ -0,0 +1,53 @@
+# ==============================================================================
+#
+#    Paybox module for OpenERP, Snesup
+#    Copyright (C) 2013 ANYBOX (<http://www.anybox.fr>)
+#
+#    This file is a part of paybox
+#
+#    paybox is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License v3 or later
+#    as published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    paybox is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License v3 or later for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    v3 or later along with this program.
+#    If not, see <http://www.gnu.org/licenses/>.
+#
+# ==============================================================================
+
+{
+    'name': 'Paybox payment acquirer',
+    'version': '1.0',
+    'sequence': 150,
+    'category': 'Custom',
+    'author': 'Odoo Community Association (OCA)',
+    'summary': "Payment acquirer: Paybox implementation",
+    'description': """Paybox payment acquirer, only useful for French organizations""",
+    'depends': [
+        'payment',
+        'l10n_fr'
+    ],
+    'data': [
+        'templates/payment_acquirer.xml',
+        'data/payment_acquirer.xml',
+        'data/email_template.xml',
+        'views/payment_acquirer.xml',
+    ],
+    'demo_xml': [
+    ],
+    'init_xml': [
+    ],
+    'css': [],
+    'test': [],
+    'installable': True,
+    'application': True,
+    'auto_install': False,
+    'license': 'AGPL-3',
+    'post_load': None,
```

Comment:
What is this key? You have post_init_hook, but not post_load. Anyway, it's not used, so please remove it.